non-transactional-dml.md

```diff
@@ -0,0 +1,294 @@
+
+{{< copyable "sql" >}}
+
+```sql
+SHOW PROCESSLIST;
+```
+
+```sql
++------+------+--------------------+--------+---------+------+------------+----------------------------------------------------------------------------------------------------+
+| Id   | User | Host               | db     | Command | Time | State      | Info                                                                                               |
++------+------+--------------------+--------+---------+------+------------+----------------------------------------------------------------------------------------------------+
+| 1203 | root | 100.64.10.62:52711 | test   | Query   | 0    | autocommit | /* job 506/500000 */ DELETE FROM `test`.`t1` WHERE `test`.`t1`.`_tidb_rowid` BETWEEN 2271 AND 2273 |
+| 1209 | root | 100.64.10.62:52735 | <null> | Query   | 0    | autocommit | show full processlist                                                                              |
++------+------+--------------------+--------+---------+------+------------+----------------------------------------------------------------------------------------------------+
+```
+
+### Terminate a non-transactional DML statement
+
+To terminate a non-transactional DML statement, you can use `KILL TIDB`. Then TiDB will cancel all batches after the batch that is currently being executed. You can get the execution result from the log.
+
+### Query the batch-dividing statement
+
+During the execution of a non-transactional DML statement, a statement is internally used to divide the DML statement into multiple batches. To query this batch-dividing statement, you can add `DRY RUN QUERY` to this non-transactional DML statement. Then TiDB will not execute this query and the subsequent DML operations.
+
+The following statement queries the batch-dividing statement during the execution of `BATCH ON id LIMIT 2 DELETE FROM t WHERE v < 6`:
+
+{{< copyable "sql" >}}
+
+```sql
+BATCH ON id LIMIT 2 DRY RUN QUERY DELETE FROM t WHERE v < 6;
+```
+
+```sql
++--------------------------------------------------------------------------------+
+| query statement                                                                |
++--------------------------------------------------------------------------------+
+| SELECT `id` FROM `test`.`t` WHERE (`v` < 6) ORDER BY IF(ISNULL(`id`),0,1),`id` |
++--------------------------------------------------------------------------------+
+1 row in set
+```
+
+### Query the statements corresponding to the first and the last batches
+
+To query the actual DML statements corresponding to the first and last batches in a non-transactional DML statement, you can add `DRY RUN` to this non-transactional DML statement. Then, TiDB only divides batches and does not execute these SQL statements. Because there might be many batches, not all batches are displayed, and only the first one and the last one are displayed.
```

Comment:
```suggestion
To query the actual DML statements corresponding to the first and the last batches in a non-transactional DML statement, you can add `DRY RUN` to this non-transactional DML statement. Then, TiDB only divides batches and does not execute these SQL statements. Because there might be many batches, not all batches are displayed, and only the first one and the last one are displayed.
```